test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -189,6 +189,9 @@ def test_pytorch_mpi_gpu(pytorch_training, ec2_connection, gpu_only, py3_only, e
     """
     if test_utils.is_image_incompatible_with_instance_type(pytorch_training, ec2_instance_type):
         pytest.skip(f"Image {pytorch_training} is incompatible with instance type {ec2_instance_type}")
+    _, framework_version = test_utils.get_framework_and_version_from_tag(pytorch_training)
+    if Version(framework_version) < Version("1.11.0"):
+        pytest.skip(f"MPI tests are skipped for PT<1.11 images")
```

Comment:
Please use fixture `pt111_and_above_only` instead of putting the skip as part of the test. By using the `pt111_and_above_only` to prevent the test function from executing, we can prevent test resources, such as EC2 instances, from being created.